packages/mdc-dialog/test/foundation.test.ts

```diff
@@ -27,8 +27,8 @@ import {cssClasses, numbers, strings} from '../constants';
 import {MDCDialogFoundation} from '../foundation';
 
 const ENTER_EVENTS = [
-  {type: 'keydown', key: 'Enter', target: {}},
-  {type: 'keydown', keyCode: 13, target: {}},
+  {type: 'keydown', key: 'Enter', target: {}, composedPath: () => [{}]},
```

Comment:
You should be able to revert these unit test changes now that the foundation handles both composedPath and event target. Our internal test on Chrome and IE11 will ensure that both paths are properly tested.